app/app.js

```diff
@@ -180,20 +183,45 @@ angular.module('app', ['angular-md5'])
         return false;
       });
 
-      if (currentCollectionIndexInCollections > 0) {
-        vm.prevCollection = collections[currentCollectionIndexInCollections - 1];
-        vm.prevCollectionFirstComic = _.find(comics, ['id', vm.prevCollection.comicIds[0]]);
-      } else {
+      //suggested revision for Klaus to check
+      // if (currentCollectionIndexInCollections > 0) {
+      //   vm.prevCollection = collections[currentCollectionIndexInCollections - 1];
+      //   vm.prevCollectionFirstComic = _.find(comics, ['id', vm.prevCollection.comicIds[0]]);
+      // } else {       
+      //   vm.prevCollection = undefined;
+      //   vm.prevCollectionFirstComic = undefined;
+      // }
+      var prevCollectionIndexInCollections;
+      var nextCollectionIndexInCollections;
+      prevCollectionIndexInCollections = currentCollectionIndexInCollections;
+      nextCollectionIndexInCollections = currentCollectionIndexInCollections;
+      prevCollectionIndexInCollections --;
+      nextCollectionIndexInCollections ++;
```

Comment:
These could maybe be one-liners by using clone like `var prevCollectionIndexInCollections = _.clone(currentCollectionIndexInCollections)--;`
I wonder if that would work

Reply:
That doesn't work for me. I get a postfix parsing error.  Also tried valueOf() with same result.

Reply:
Cool it was worth a shot